helm/chart-operator/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for the chart-operator
+name: chart-operator
+version: 0.9.0
```

Comment:
```suggestion
version: [[ .Version ]]
```

Reply:
Thanks @kopiczko yes since this is the chart for the app catalog we should use this.